features/auth/signup/Signup.tsx

```diff
@@ -178,21 +179,28 @@ export default function Signup() {
 
   useEffect(() => {
     (async () => {
-      if (urlToken) {
+      if (activationUID && activationToken) {
         setLoading(true);
         try {
-          authActions.updateSignupState(
-            await service.auth.signupFlowEmailToken(urlToken)
-          );
+          await service.auth.activateUser(activationUID, activationToken);
+          authActions.updateSignupState({
+            needBasic: false,
+            needAccount: true,
+            needAcceptCommunityGuidelines: true,
+            needFeedback: true,
+            needVerifyEmail: false,
+            flowToken: "token",
+          });
```

Comment:
```suggestion
```
Copied from the comment I made on the `pages/activate/[uid]/[token]/index.tsx` file:
> Once Hamish gets around adding the step computation logic, I think it'd make less sense to manipulate the "flow state" in the frontend code ourselves.

That's what I was referring to and should have written it here in the first place. The fact you wrote a dummy token for `flowToken` here suggests it's not so useful/doesn't work for the new API flow.